api/logging_api/routes/LedSign.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+const axios = require('axios');
+const router = express.Router();
+const { LED_SIGN_URL } = require('../../config/config.js');
+const { OK, BAD_REQUEST } = require('../../util/constants').STATUS_CODES;
+const AWS = require('aws-sdk');
+const { ledSqsKeys } = require('../../config/config.json');
+
+creds = new AWS.Credentials(ledSqsKeys.CLIENT_ID, ledSqsKeys.CLIENT_SECRET);
+AWS.config.update({
+  region: 'us-west-1',
+  endpoint: LED_SIGN_URL,
+  credentials: creds
+});
+
+const sqs = new AWS.SQS({ apiVersion: '2012-11-05' });
+const queueUrl =
+'https://sqs.us-west-2.amazonaws.com/'
++ ledSqsKeys.ACCOUNT_ID + '/' + ledSqsKeys.QUEUE_NAME;
+
+// send as part of request body which isn't present in url
+router.use(express.json());
+
+router.get('/healthCheck', (req, res) => {
+  axios.get(LED_SIGN_URL + 'healthCheck')
```

Comment:
where is this sending the request to? i thought your backend just pushes messages to sqs

Reply:
Health check is necessary in order to run the LED Sign. We should make a future PR in order to make Health Check possible using SQS.

Reply:
yes, and for now you should just be returning 200 in this function. remember that the computer does not have access to the LED sign. if it did then we wouldn't be using SQS at all